packages/block-editor/src/components/block-list/use-multi-selection.js

```diff
@@ -229,19 +229,19 @@ export default function useMultiSelection( ref ) {
 	}, [ onSelectionChange, stopMultiSelect ] );
 
 	// Only clean up when unmounting, these are added and cleaned up elsewhere.
-	useEffect(
-		() => () => {
-			const { ownerDocument } = ref.current;
```

Comment:
It's strange that the ref could be gone here. I can't reproduce with a simplified case: https://codepen.io/iseulde/pen/rNegKdy

Reply:
I also created a test where I unmount the whole BlockList component if you resize the window, and I don't see any error in `master`...